hbase-server/src/main/java/org/apache/hadoop/hbase/master/HMaster.java

```diff
@@ -1018,7 +1018,7 @@ private void finishActiveMasterInitialization(MonitoredTask status)
     RegionState rs = this.assignmentManager.getRegionStates().
         getRegionState(RegionInfoBuilder.FIRST_META_REGIONINFO);
     LOG.info("hbase:meta {}", rs);
-    if (rs != null && rs.isOffline()) {
+    if ((rs == null) || (rs != null && rs.isOffline())) {
```

Comment:
When it is a new meta replica config, there is no meta region server node at zookeeper. In the old way, it will create a RegionState with offline state. In this new way, there will be no RegoinState created. So that is why rs == null is added.

Reply:
ok